webapp/src/components/router_showcase/router_showcase.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+    Switch,
+    Route,
+    useLocation,
+} from 'react-router-dom';
+import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+import {useSelector} from 'react-redux';
+
+import {id} from 'src/manifest';
+
+export default function RouterShowcase() {
+    const currentTeam = useSelector(getCurrentTeam);
+    const query = useQuery();
+
+    return (
+        <Switch>
+            <Route path={`/${currentTeam.name}/${id}/teamtest/subpath`}>
+                <h3>{'Hello, Subpath Component!'}</h3>
+            </Route>
+            <Route path={`/${currentTeam.name}/${id}/teamtest/subpath-with-query`}>
+                <h3>{'Hello, Subpath with Query!'}</h3>
+                <p>{`The search-value in the query string is "${query.get('search-value')}"`}</p>
```

Comment:
This is a great example, but also a bit dangerous, because I can now create a link with a query value of "Mattermost sucks!" and you'll see that in the screen :joy: Can we add some simple logic here depending on the `search-value`? This is just an example, but maybe something like showing something like "on", "off" or "undefined" depending on whether the search-value is "on", "off" or anything else. What do you think?